frontend_server/lib/server.dart

```diff
@@ -37,6 +37,7 @@ ArgParser _argParser = new ArgParser(allowTrailingOptions: true)
       help: 'Path to file byte store used to keep incremental compiler state.'
           ' If omitted, then memory byte store is used.',
       defaultsTo: null)
+  ..addFlag('aot', help: 'Run compiler in AOT mode', defaultsTo: false)
```

Comment:
Perhaps worth elaborating a little what are the consequences of running compiler in AOT (vs JIT?) mode. I mean the end result is just a kernel file, so it's not obviously clear why can't use kernel file produced with aot=true in VM JIT setting.

Reply:
Added more details to the help message.